### ISSUE TITLE
Full localization support

### ISSUE THREAD

**Issue by Adaptivity** [MachineMuse/MachineMusePowersuits#621]
Still a lot of stuff is hardcoded. Including GUI elements and item tooltips. Please, only close this when *everything* will be translatable.

Continuation of #516

**Comment by MachineMuse** [comment]
I don't even know what's still missing

**Comment by Adaptivity** [comment]
A lot of stuff. Tons of... I'll create a list for you :)

**Comment by Adaptivity** [comment]
Added to the first post. It just starting. I'll search for more stuff tomorrow :)

**Comment by MachineMuse** [comment]
ok, thanks.

You're right that some of this would require a rework of those systems so I'm not really thrilled about that idea. The models especially were designed to be something that players can contribute to and have custom ones easily, but nobody did :(

**Comment by aionys** [comment]
Models?

**Comment by Xen1290** [comment]
Models?
An updated in-game wiki book or an updated wiki for MPS is needed once MPS is released.
Didn't know that feature....

**Comment by MachineMuse** [comment]
yeah some documentation would probably have helped with that. ^^' oh well.

**Comment by Adaptivity** [comment]
@MachineMuse After thinking a bit, it's maybe not required to rewrite everything. I suggest you this system: 1) create folders armor and modelspec. 2) move .xml files to these folders and rename them to language codes (en_US.xml). 3) create in these folders folder "Custom" where custom models can be placed. Here an example how to make .xml files translatable: https://github.com/SlimeKnights/TinkersConstruct/commit/fb4c0ddf9e05d03fa871300eb9df91524d3610df It should be different from yours but it's should be possible.

**Comment by MachineMuse** [comment]
the main problem is that the name is stored in the nbt tags, so if these things are changed, it would appear incorrectly for people using different locales on the same server.

The simplest solution would be to just make them display in the GUI with locales, but the locale data would have to be added to the main file in the lang folder as far as I know, and that wouldn't support custom models in any case.

**Comment by Adaptivity** [comment]
Then maybe you can separate supported models from custom? So, supported will be stored in main lang file but custom will stay in .xml.

**Comment by Joccob** [comment]
So far, I guess the only thing left that cannot be localized is item tooltips. I hope this can be finished in the future versions. @MachineMuse

**Comment by lehjr** [comment]
Most of this is actually done in 1.12.2 now.
